aliskin/assignment4/tests.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from map_class import Map
+from count_islands import count_islands
+
+class TestEmptyMap(unittest.TestCase):
+    def setUp(self):
+        self.map_ = Map([[False for i in range(100)] for j in range(100)])
+
+    def test_empty_map(self):
+        self.assertEqual(count_islands(self.map_), 0)
+
+class TestFullMap(unittest.TestCase):
+    def setUp(self):
+        self.map_ = Map([[True for i in range(100)] for j in range(10)])
+
+    def test_full_map(self):
+        self.assertEqual(count_islands(self.map_), 1)
+
+class TestChessMap(unittest.TestCase):
+    def setUp(self):
+        size = 100
+        map_items = [[False for i in range(size)] for j in range(size)]
+        for i in range(size):
+            for j in range(size):
+                if (i + j) % 2 == 0:
+                    map_items[i][j] = True
+        self.map_ = Map(map_items)
+
+    def test_chess_map(self):
+        self.assertEqual(count_islands(self.map_), self.map_.x_max * self.map_.y_max // 2)
+
+class TestInnerSea(unittest.TestCase):
+    def setUp(self):
+        self.map_ = Map([[True, True, True, True],
```

Comment:
nit: please fix indentation so all the rows of the map are aligned, in this way it's easier to have an idea of the map :)